packages/react/test/client-generator.test.ts

```diff
@@ -1,17 +1,14 @@
 import { expectTypeOf, it } from "vitest";
 import * as z from "zod";
 
+import type { UploadFileResponse } from "uploadthing/client";
 import { createUploadthing } from "uploadthing/server";
 
 import { generateReactHelpers } from "../src/hooks";
 
-function ignoreErrors(fn: () => void) {
-  try {
-    fn();
-  } catch {
-    // no-op
-  }
-}
+const doNotExecute = (_fn: (...args: any[]) => any) => {
```

Comment:
I just got tired of the rules of hook warnings since we called the hook outside a component, and since we don't test any runtime code, it made sense to turn the ignoreErrors to a doNotExecute. Ofc we can go the extra mile and use something like `renderHook` from `@testing-library/react` but i dont think we need that at this point.